CMake/macros.cmake

```diff
@@ -19,26 +19,17 @@ INCLUDE("${PROJECT_SOURCE_DIR}/CMake/test_labels.cmake")
 
 # Function to copy a directory
 FUNCTION( COPY_DIRECTORY SRC_DIR DST_DIR )
-    EXECUTE_PROCESS( COMMAND ${CMAKE_COMMAND} -E copy_directory "${SRC_DIR}" "${DST_DIR}" )
+    file(COPY ${SRC_DIR} DESTINATION ${DST_DIR})
```

Comment:
This changed behavior. 
COPY_DIRECTORY(folder_a folder_b)
is expect to produce a new folder_b not folder_b/folder_a